src/MCPClient/lib/clientScripts/emailFailReport.py

```diff
@@ -20,166 +20,205 @@
 # @package Archivematica
 # @subpackage archivematicaClientScript
 # @author Joseph Perry <joseph@artefactual.com>
+
 import smtplib
+import sys
 from email.mime.multipart import MIMEMultipart
 from email.mime.text import MIMEText
-from optparse import OptionParser
+import argparse
 from lxml import etree
 
 import django
 django.setup()
 # dashboard
+from django.db import connection
 from django.contrib.auth.models import User
+from main.models import Job, Report
 
 # archivematicaCommon
 from custom_handlers import get_script_logger
 import databaseInterface
-from externals.HTML import HTML 
+from externals.HTML import HTML
+
+
+logger = get_script_logger('archivematica.mcp.client.emailFailReport')
+
+
+COLORS = {
+    'Completed successfully': '#dff0d8',
+    'Failed': '#f2dede',
+    'default': 'yellow',
+}
 
-def getEmailsFromDashboardUsers():
-    return [u[0] for u in User.objects.filter(is_active=True).values_list('email').exclude(email__in=['demo@example.com', ''])]
+def get_emails_from_dashboard_users():
+    users = User.objects.filter(is_active=True).values_list('email').exclude(email__in=['demo@example.com', ''])
+    return [u[0] for u in users]
```

Comment:
Tip: This can be done with `flat=True`, eg `return User.objects.filter(is_active=True).values_list('email', flat=True).exclude(email__in=['demo@example.com', ''])`